components/director/Makefile

```diff
@@ -2,6 +2,15 @@ APP_NAME = compass-director
 APP_PATH = components/director
 BUILDPACK = eu.gcr.io/kyma-project/test-infra/buildpack-golang:go1.18
 SCRIPTS_DIR = $(realpath $(shell pwd)/../..)/scripts
+DEPLOYMENT_DIRECTOR_NAME="compass-director"
```

Comment:
what is the motivation behind this

Reply:
No need, it was a leftover. I will optimise this, by using the values directly instead variables.